packages/ui/form/PhoneInput.tsx

```diff
@@ -58,14 +59,21 @@ function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputP
 }
 
 const useDefaultCountry = () => {
-  const [defaultCountry, setDefaultCountry] = useState("us");
+  const [defaultCountry, setDefaultCountry] = useState("");
   trpc.viewer.public.countryCode.useQuery(undefined, {
     refetchOnWindowFocus: false,
     refetchOnReconnect: false,
     retry: false,
-    onSuccess: (data) => {
-      if (isSupportedCountry(data?.countryCode)) {
-        setDefaultCountry(data.countryCode.toLowerCase());
+    onSuccess: async (data) => {
+      try {
+        if (isSupportedCountry(data?.countryCode)) {
+          setDefaultCountry(data.countryCode.toLowerCase());
+        } else {
+          const response = await axios.get("https://ipapi.co/json/");
```

Comment:
This is a paid API(https://ipapi.co/pricing/) not sure if we should use this or not? cc: @PeerRich @emrysal

Reply:
We are using this api to get the location more precisely, but we can navigator.language instead as it's a secundary way to grab the countryCode

Reply:
@gitstart-calcom ok just use navigator.language for now.